pytest_splunk_addon/standard_lib/data_models/Intrusion_Detection.json

```diff
@@ -99,12 +109,12 @@
         },
         {
           "name": "user",
-          "type": "optional",
+          "type": "required",
```

Comment:
this will be blank in many cases, think about an IPS that is on the perimeter of the network protecting a public website. thousands of IPs from the internet will hammer the site day in and day out triggering IPS alerts yet have no user attached.

Reply:
Ok, makes sense. I looked at the recent AWS guardduty events and i saw the users there. However that could be not a common thing for the IDS in general. @harshilgajera-crest please revert to optional. I corrected the sheet for consistency too.

Reply:
Done